tests/README.md

```diff
@@ -114,42 +144,48 @@ To do that run test(s) with `--pdb` flag:
 make tests args="[TESTFILE]::[TESTNAME] --pdb"
 ```
 
-Alternatively you can add `breakpoint()` before the failing assertion
-and manually invoke the single test.
+Alternatively you can add `breakpoint()` before the failing assertion and manually invoke the single test.
+
+### Container Shell
+
+You can also drop into a shell in the ` tests` container via `docker compose run --entrypoint=sh tests`, and from there run `pytest` manually.
```

Comment:
should we add `make tests/sh` or something to fasciliate that? I think you need to do that pretty often

Reply:
I do do it pretty often, a `make` command for this would be great! Let's do it in another pr and update the doc there, I'd like to keep this one just docs without code changes.